dashboard/app/api.go

```diff
@@ -40,6 +39,7 @@ var apiHandlers = map[string]APIHandler{
 }
 
 var apiNamespaceHandlers = map[string]APINamespaceHandler{
+	"job_poll":            apiJobPoll,
```

Comment:
Why do we need to make this per-namespace request?
If we do this we also need to update syz-ci because it sends job_poll requests using client that is not attached to any namespace, so these requests will start failing.
But syz-ci sends a list of managers that it supports and that should be enough, no?

Reply:
I see, I assumed that all clients are associated with a namespace and so I thought that we can filter the bugs using that namespace information(so that more relevant bugs are fetched). I'll remove this change.